contacts/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Add test cases here as well. At least 1 test case for each feature.

Reply:
i added at contacts/tests/app_test.py

Reply:
check now